### PR TITLE
Fix environment-based config and permissions

### PR DESCRIPTION
**Pull request by ericmason**
- Fix a bug where `FRESHCLAM_CONF_FILE` was used instead of `CLAMD_CONF_FILE`
- Change ownership of /etc/clamav to clamav so it doesn't fail to create tempfiles there when using config environment variables

### CODE REVIEW

File: debian/buster/Dockerfile [hunk 1]
```diff
@@ -48,7 +48,7 @@ COPY bootstrap.sh /
 # port provision
 EXPOSE 3310
 
-RUN chown clamav:clamav bootstrap.sh check.sh envconfig.sh /etc/clamav/clamd.conf /etc/clamav/freshclam.conf && \
+RUN chown clamav:clamav bootstrap.sh check.sh envconfig.sh /etc/clamav /etc/clamav/clamd.conf /etc/clamav/freshclam.conf && \
```
Comment (mko-x) [inline debian/buster/Dockerfile:51]: Why did you remove /etc/clamav as folder from chown?

**Review by mko-x — APPROVED** [review]
Thank you

File: debian/buster/Dockerfile (hunk 1 above)
Comment (ericmason) [reply debian/buster/Dockerfile:51]: I added /etc/clamav. It was owned by root previously and creating the backup config file and the sed backups were failing, so you couldn't use config variables or provide your own config file.
